README.md

```diff
@@ -1 +1,24 @@
 # Operator Marketplace
+
+## Deploying the Marketplace Operator
+### Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Be logged into an OpenShift cluster as a user with the RBAC permissions required to run this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+2. Have created the CatalogSourceConfig custom resource defined within the [Operator Lifecycle Manager](https://github.com/operator-framework/operator-lifecycle-manager) project.
+3. Have authorization to pull the Marketplace Operator Image from quay.io/redhat/marketplace, you can make a request to join the “RedHat” organization by following the steps outlined [here](https://mojo.redhat.com/docs/DOC-1167707)
+4. Update the deploy/marketplace-pull-secret.yaml file with your secret.
+
+#### No Quay Access? Create the Marketplace Operator in your own public repo
+1. Update the deploy/operator.yaml to pull from a public registry. You should update the spec.template.spec.containers[0].image value with the <IMAGE DESTINATION> value. Example: `docker.io/awgreene/marketplace-operator`
```

Comment:
inline highlights to be consistent with the rest - `deploy/operator.yaml ` and `pec.template.spec.containers[0].image`

Reply:
Highlights are now consistent.